### PR TITLE
Fix linter warnings

### PR DESCRIPTION
**Pull request by tituspijean**
## Problem
- App is level 4 due to various linter warnings, e.g. `ask` strings left in manifest and numerous Composer warnings.

### CODE REVIEW

**Comment by tituspijean** [comment]
!testme

**Comment by tituspijean** [comment]
!testme

**Comment by tituspijean** [comment]
!testme

**Comment by tituspijean** [comment]
Sorry for breaking you, manifest.
!testme

**Comment by tituspijean** [comment]
!testme

**Comment by tituspijean** [comment]
I... will... succeed... some... day...
!testme

**Review by ericgaspar — COMMENTED** [review]
I think you can delete those lines as [ynh_legacy_permissions_delete_all](https://github.com/YunoHost/yunohost/blob/adc83b4c9c2c30e9ef75f3609c538b646f91f1db/data/helpers.d/permission#L399 ) should remove those permissions. https://github.com/YunoHost-Apps/flarum_ynh/blob/91342f4ac236130228a4cdd7f021a357ebe049ce/scripts/upgrade#L70-L83

File: manifest.json [hunk 1]
```diff
@@ -7,15 +7,15 @@
...
+    "version": "0.1.0.14~ynh3",
     "url": "http://flarum.org/",
     "license": "MIT",
     "maintainer": {
         "name": "tituspijean",
         "email": "tituspijean@outlook.com"
     },
     "requirements": {
-        "yunohost": ">= 3.8.1"
+        "yunohost": ">= 4.1"
```
Comment (ericgaspar) [inline manifest.json]: ... and bump requirements to 4.1.2 if you are using `ynh_legacy_permissions_delete_all` https://github.com/YunoHost/yunohost/blob/adc83b4c9c2c30e9ef75f3609c538b646f91f1db/data/helpers.d/permission#L398

**Comment by tituspijean** [comment]
!testme

**Comment by tituspijean** [comment]
@ericgaspar we should be all good now. ;)

**Review by ericgaspar — APPROVED** [review]

**Pull request merged**